drivers/libhid.c

```diff
@@ -456,9 +456,7 @@ int HIDSetDataValue(hid_dev_handle_t udev, HIDData_t *hiddata, double Value)
 	}
 
 	/* flush the report buffer (data may have changed) */
-	for (i=0; i<256; i++) {
-		reportbuf->ts[i] = 0;
-	}
+	memset(reportbuf->ts, '\0', sizeof(reportbuf->ts)/sizeof(reportbuf->ts[0]));
```

Comment:
Dividing by `sizeof(...ts[0])` doesn't seem right - each element of `ts` is a `time_t`, and the len parameter of memset() is in bytes (despite the value being an int).